CHANGELOG-1.1.md

```diff
@@ -0,0 +1,243 @@
+# Release notes for 1.1.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.0.1
+
+## Changes by Kind
+
+### Feature
+
+- Added leader election health check at the metrics port + path "/healthz/leader-election".
+  - klog/v2 is used for logging
+  - process_start_time metric is now reported
+  - `metrics-address` flag is deprecated and replaced by `http-endpoint`, which enables handlers from both metrics manager and leader election health check. ([#129](https://github.com/kubernetes-csi/external-resizer/pull/129), [@verult](https://github.com/verult))
+
+### Bug or Regression
+
+- Changing distroless image back to multiarch ([#120](https://github.com/kubernetes-csi/external-resizer/pull/120), [@namrata-ibm](https://github.com/namrata-ibm))
```

Comment:
I think we can remove this release note because it was only a temporary thing that never got released.

Reply:
removed.